sparse_strips/vello_hybrid/Cargo.toml

```diff
@@ -17,7 +17,7 @@ workspace = true
 [dependencies]
 bytemuck = { workspace = true, features = ["derive"] }
 thiserror = { workspace = true }
-vello_common = { workspace = true, features = ["std"] }
+vello_common = { workspace = true, features = ["std", "text"] }
```

Comment:
Hmm, this is a bit of a surprise

Reply:
I don't think we should revert this (at least not without discussion). But I do think that this will require significant thinking work to behave nicely with Vello API.